src/test/emulators/extensionsEmulator.spec.ts

```diff
@@ -0,0 +1,81 @@
+import { expect } from "chai";
+
+import { ExtensionsEmulator } from "../../emulator/extensionsEmulator";
+import { EmulatableBackend } from "../../emulator/functionsEmulator";
+import * as planner from "../../deploy/extensions/planner";
+
+describe("Extensions Emulator", () => {
+  describe("toEmulatableBackends", () => {
+    let previousCachePath: string | undefined;
+    beforeEach(() => {
+      previousCachePath = process.env.FIREBASE_EXTENSIONS_CACHE_PATH;
+      process.env.FIREBASE_EXTENSIONS_CACHE_PATH = "./extensions";
```

Comment:
nit* It looks like we expect each test case to set ^ instead, so this default value is never used. That is fine - I was just a bit confused.

Reply:
Nope, this was sloppy from me  - I was moving this bit around while I was trying to debug this test & forgot to clean it up.